infrastructure/sentry-subgraph/src/pool-factory.ts

```diff
@@ -175,7 +175,9 @@ export function handleUnstakeKeys(event: UnstakeKeys): void {
     }
 
     // Update the Users Pool Stake 
-    const poolStakeId = event.params.pool.toHexString() + "_" +  event.params.user.toHexString();
+    const poolAddress = event.params.pool.toHexString().toLowerCase();
+    const userAddress = event.params.user.toHexString().toLowerCase();
+    const poolStakeId = poolAddress + "_" +  userAddress;
```

Comment:
This does not seem right. Could this have been fixed with adding the poolStake to createPool ?

Reply:
Redeployed with `toLowerCase` removed to see if we get any errors.

[New Deployment](https://subgraphs.alchemy.com/subgraphs/4740/versions/30454)